main.tf

```diff
@@ -34,6 +34,14 @@ resource "aws_s3_bucket_versioning" "this" {
   }
 }
 
+resource "aws_s3_bucket_ownership_controls" "this" {
+  bucket = aws_s3_bucket.this.id
+
+  rule {
+    object_ownership = "BucketOwnerEnforced"
+  }
+}
+
```

Comment:
I notice the official S3 module has an explicit depends on statement.  I wonder if we should add it here too?  https://github.com/terraform-aws-modules/terraform-aws-s3-bucket/blob/master/main.tf#L747